Gruntfile.js

```diff
@@ -383,6 +383,23 @@ module.exports = function (grunt) {
             src: ['**', '!hmda-pilot.zip']
           }
         ]
+      },
+      'codedeploy': {
```

Comment:
I think the approach you have is the correct one here. Good call on the separation.